CakePHP/Sniffs/NamingConventions/ValidFunctionNameSniff.php

```diff
@@ -114,7 +114,7 @@ protected function processTokenWithinScope(PHP_CodeSniffer_File $phpcsFile, $sta
 			if (substr($className, -4) === 'Task') {
 				return;
 			}
-		} elseif ($isPrivate === true) {
+		} elseif ($isPrivate === true && strpos($phpcsFile->getFilename(), '/lib/cake') === true) {
```

Comment:
hmm, sure that is weird.

This was in version 2.0.3 were I copied it from:
```php
elseif ($isPrivate === true) {
	if (substr($methodName, 0, 2) !== '__') {
		$error = 'Private method name "%s" must be prefixed with 2 underscores';
		$phpcsFile->addError($error, $stackPtr, 'PrivateNoUnderscore', $errorData);
		return;
	} else {
		$filename = $phpcsFile->getFilename();
		if (strpos($filename, '/lib/Cake/') === true) {
			$warning = 'Private method name "%s" in CakePHP core is discouraged';
			$phpcsFile->addWarning($warning, $stackPtr, 'PrivateMethodInCore', $errorData);
		}
	}
}
```

Would it work for you if we use `==` ?